packages/terra-site/src/examples/form-checkbox/CheckboxFieldIndex.jsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
+import React from 'react';
+import DocTemplate from 'terra-doc-template';
+import ReadMe from 'terra-form-checkbox/docs/CheckboxField.md';
+
+// Component Source
+import CheckboxFieldSrc from '!raw-loader!terra-form-checkbox/src/CheckboxField';
+
+// Examples
+import DefaultCheckboxField from './checkbox_field_examples/DefaultCheckboxField';
+import DefaultCheckboxFieldSrc from '!raw-loader!./checkbox_field_examples/DefaultCheckboxField.jsx';
```

Comment:
The source code doesn't seem to be rendering in the browser, try dropping the `.jsx` from this line.